addons/website_blog/views/website_blog_templates.xml

```diff
@@ -230,23 +223,23 @@ list of filtered posts (by date or tag).
             </div>
         </section>
 
-        <section id="o_wblog_post_main" t-attf-class="container pt-4 pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
+        <section id="o_wblog_post_main" t-attf-class="pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
```

Comment:
Maybe there should not be `<section/>` in the base layout as they will be used for the actual content